charts/ceph-csi-cephfs/values.yaml

```diff
@@ -30,6 +30,19 @@ serviceAccounts:
 #       netNamespaceFilePath: "{{ .kubeletDir }}/plugins/{{ .driverName }}/net"
 csiConfig: []
 
+# Configuration for the encryption KMS
+# Ref: https://github.com/ceph/ceph-csi/blob/devel/docs/deploy-cephfs.md#cephfs-volume-encryption
```

Comment:
this line seems to be too long ([ci job](https://github.com/ceph/ceph-csi/actions/runs/8514964926/job/23336366170?pr=4531#step:3:1116))

Reply:
Is it okay for me to break the link into two lines? Or perhaps I should add lint silencer for this line? What is the suggestion?

Reply:
Adding a lint silencer for that line is fine. Breaking the URL over multiple lines isn't that clean.

Reply:
This line directly above the URL should work:
```
# yamllint disable-line rule:line-length
```